cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -24,6 +24,12 @@ import (
 	"github.com/golang/glog"
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/util"
+	"github.com/pkg/errors"
+)
+
+var (
+	poolTypeMap         = map[string]string{"mirrored": "mirror", "raidz": "raidz", "raidz2": "raidz2"}
```

Comment:
will `registeredPools` be a better name?
what about `supportedPools`?

**NOTE: Name based on what it provides & not what it contains.**

Reply:
what about `poolTypeMap`->`poolTypeCommand` since it's command mapped with poolType.

Reply:
`poolTypeCommandMap` can be good